pkg/generate/app/app.go

```diff
@@ -576,3 +581,12 @@ func LabelsFromSpec(spec []string) (map[string]string, []string, error) {
 	}
 	return labels, remove, nil
 }
+
+// addGeneratedByAnnotation adds openshift.io/generatedby to objects generated by
+// the new-app
+func addGeneratedByAnnotation(meta *kapi.ObjectMeta) {
+	if len(meta.Annotations) == 0 {
+		meta.Annotations = map[string]string{}
+	}
+	meta.Annotations["openshift.io/generatedby"] = "OpenShiftNewApp"
```

Comment:
Web console currently uses `openshift.io/generatedby`, so we'd want to change it there, too.